src/main/java/blusunrize/immersiveengineering/common/util/compat/computers/generic/impl/PoweredMBCallbacks.java

```diff
@@ -0,0 +1,16 @@
+package blusunrize.immersiveengineering.common.util.compat.computers.generic.impl;
+
+import blusunrize.immersiveengineering.common.blocks.generic.PoweredMultiblockTileEntity;
+import blusunrize.immersiveengineering.common.util.compat.computers.generic.Callback;
+
+public class PoweredMBCallbacks extends Callback<PoweredMultiblockTileEntity<?, ?>>
+{
+	public static final PoweredMBCallbacks INSTANCE = new PoweredMBCallbacks();
+
+	public PoweredMBCallbacks()
+	{
+		addAdditional(EnergyCallbacks.INSTANCE);
+		addAdditional(MultiblockCallbacks.INSTANCE);
+	}
+	//TODO general access to recipes?
```

Comment:
Hmm. Unsure on that. Seems very potent.

Reply:
I was talking about access to the current recipe, so you could e.g. make a display "currently converting plant oil and ethanol to biodiesel at a ratio of 2:2:4". That seems reasonable from a potency perspective to me. But I've essentially put that on the list of "implement this once I find an elegant approach for it".

Reply:
Ah, I see